cmd/subctl/aws.go

```diff
@@ -53,8 +53,8 @@ var (
 	awsCleanupCmd = &cobra.Command{
 		Use:   "aws",
 		Short: "Clean up an AWS cloud",
-		Long: "This command cleans up an OpenShift installer-provisioned infrastructure (IPI) on AWS-based" +
-			" cloud after Submariner uninstallation.",
+		Long: "This command cleans up an OpenShift installer-provisioned infrastructure (IPI) on " +
```

Comment:
Am I missing something. Where is the duplication reduced in this PR?

Reply:
The partial text " _cloud after Submariner uninstallation._" for wrapping. I just refactored the wrapped part so it n o longer appears as a dupe.